### DISCUSSION TITLE
How to make policy fail when bad jwt token value is given to io.jwt.decode_verify ?

### DISCUSSION THREAD

**Discussion by DamienDelporte** [open-policy-agent/community#497] [OPA and Rego]
I managed to make it work with a valid JWT token value :ok_hand:

But when not giving a valid JWT token like "not.a.jwt" to io.jwt.decode_verify() I don't have any idea how to make a denial :/
https://play.openpolicyagent.org/p/gZ8hF22hIF

Thank you for your suggestions :pray:

**Comment by anderseknert** [comment]
You could add another `deny` rule for that case:

```rego
deny["provided token not a JWT"] {
    not io.jwt.decode(input.message)
}
```

**Reply by DamienDelporte** [reply]
Awesome, that was it ! Thank you so much !
